test/cpp/jit/backend_with_compiler.cpp

```diff
@@ -0,0 +1,111 @@
+#include <torch/csrc/jit/backends/backend.h>
+
+namespace torch {
+namespace jit {
+
+namespace {
+std::vector<std::string> parseBlob(const std::string& blob) {
```

Comment:
Since there could be many "blobs", maybe rename "parseBlob" to something more specific (e.g. parseMethodHandle or something like that).